config/config.go

```diff
@@ -15,7 +15,7 @@ const (
 	DefaultC2TBatchSubmissionGasLimit       = uint64(500_000)
 	DefaultMMBatchSubmissionGasLimit        = uint64(550_000)
 	DefaultDepositBatchSubmissionGasLimit   = uint64(220_000)
-	DefaultTransitionDisputeGasLimit        = uint64(4_100_000)
+	DefaultTransitionDisputeGasLimit        = uint64(4_500_000)
```

Comment:
I haven't figured out a principled way to set this number. The update to hubble-contracts pushed up our gas costs and caused an e2e test to fail. Bumping this limit fixes the test but before we decentralize the sequencer it would be prudent to go back and make sure this is truly an upper bound. If it's too low then we'll be unable to dispute incorrect transitions.